lib/matplotlib/patches.py

```diff
@@ -3218,23 +3218,52 @@ class _Curve(_Base):
         """
         A simple arrow which will work with any path instance. The
         returned path is the concatenation of the original path, and at
-        most two paths representing the arrow head at the begin point and the
-        at the end point. The arrow heads can be either open or closed.
+        most two paths representing the arrow head or bracket at the begin
+        point and the at the end point. The arrow heads can be either open
+        or closed.
         """
 
-        beginarrow = endarrow = False  # Whether arrows are drawn.
+        beginarrow_curve = endarrow_curve = False  # Whether arrows are drawn.
```

Comment:
Should I do the deprecation dance or leave it as `beginarrow` and `endarrow`?

Reply:
I'm not sure having two attributes makes sense, actually. If you set both to `True`, then does drawing both _types_ on top of each other make sense? If not, it would be better to have one option that takes a string, and warns on the old boolean values, but does what it used to.

Reply:
As it is right now, if both the `beginarrow` and `beginarrow_bracket` are `True`, the `beginarrow` ('<-') one would be drawn. But I agree that passing it as an string would make more sense.